src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/ApplicationService.kt

```diff
@@ -8,7 +8,7 @@ import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.ProbationOffi
 import uk.gov.justice.digital.hmpps.approvedpremisesapi.model.AuthorisableActionResult
 import uk.gov.justice.digital.hmpps.approvedpremisesapi.model.ValidatableActionResult
 import uk.gov.justice.digital.hmpps.approvedpremisesapi.model.ValidationErrors
-import uk.gov.justice.digital.hmpps.approvedpremisesapi.model.singleValidationErrorOf
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.model.validated
```

Comment:
Am I being dim? Where is this model defined?

Reply:
Yes I am 😆

Reply:
This is one of the reasons I'm on the fence about whether this is better than after the first commit.  Perhaps a bit too confusing at first glance?